### ISSUE TITLE
Windows Installation fails on Ruby on WSL step

### ISSUE THREAD

**Issue by colorfusion** [thepracticaldev/dev.to#8186]
**Describe the bug**

Setting up DEV on Windows 10 fails at the step `Ruby on WSL` when `$(cat .ruby-version)` is called as `.ruby-version` does not exist.

**To Reproduce**

1. Follow [Installation for Windows](https://docs.dev.to/installation/windows/) in DEV docs
2. Run line 11 / 12 for installing ruby
3. Observe that `.ruby-version` file does not exist.

**Expected behavior**

`$(cat .ruby-version)` should evaluate to a ruby version allowing `rbenv` to work.

**Screenshots**

![image](https://user-images.githubusercontent.com/1024128/83330865-9ed0df80-a2c4-11ea-9557-9ba6543194c7.png)

**Desktop (please complete the following information):**

- OS: Windows 10
- Browser: Firefox Browser
- Version: 76.0.1 (64-bit)

**Smartphone (please complete the following information):**

- Device:
- OS:
- Browser:
- Version:

**Additional context**

**Comment by rhymes** [comment]
@colorfusion the documentation is not super clear on this part I guess, but you have to be in the repository's clone folder for that step to work. This is the `.ruby-version` file: https://github.com/thepracticaldev/dev.to/blob/master/.ruby-version

**Comment by rhymes** [comment]
It's weird, the documentation assumes you've already downloaded the repo even though it tells you to download the repo _at the end_ of the install process

**Comment by Amorpheuz** [comment]
I think replacing the line `rbenv install $(cat .ruby-version)` with `rbenv install $(wget -O- -q https://raw.githubusercontent.com/thepracticaldev/dev.to/master/.ruby-version)` should solve the issue of not having the repo downloaded. Can make the PR if the solution seems fit!

**Comment by rhymes** [comment]
Could that be done only in WSL mode?

cc @maestromac

**Comment by colorfusion** [comment]
If the `.ruby-version` on your local branch is different from remote's master for some reasons, wouldn't that cause some compatibility issues if you install based on that?

**Comment by colorfusion** [comment]
I would assume referring to the local's copy makes more sense since the application is supposed to be working at that point when you checked out the branch.

**Comment by rhymes** [comment]
@colorfusion yeah, that's a valid point. downloading master's version of the file could mean that the local copy could install the wrong version at times

**Comment by colorfusion** [comment]
@rhymes That reminds me, after checking out the repo and installing ruby. It seems like page still doesn't start up properly even after running, `bin/setup`. Should that be written in a new issue, or should we rename this issue to a more general name (DEV installation on Windows not working)?

**Comment by rhymes** [comment]
I believe they should be two separate issues, thanks @colorfusion!

**Comment by Amorpheuz** [comment]
That does work in **WSL 2** for me. I think since that section is under `prerequisites` for `Installation`, it is okay to obtain the version from the repo. I didn't want to change the flow of the docs since then it does become different from the flow of the other two docs (Linux and Mac). I think a person who will be upgrading and modifying ruby versions will be familiar with the concept. 

Another Idea, I can think of is to switch back to the old way which if I remember correctly asked installers to pick the ruby version from the ruby badge on the README and link [Digital Ocean's Guide](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-18-04) for the same.

**Comment by colorfusion** [comment]
Oh, I meant that running `bin/setup` failed in some steps even in WSL. I've already fixed it on my end, and added another issue #8446 for future reference.

I believe a quick fix to the documentation would be just to shift the checkout of the DEV repo to the prerequisites, that way you can adjust the first step for installing ruby to `cd dev.to` instead.

**Comment by Amorpheuz** [comment]
Ah the first para was meant for Rhymes for
> Could that be done only in WSL mode?

I should really start using quotes, keep forgetting about them. 😅

Will wait on a reply from the core team on what's the best route before making a PR!

**Comment by maestromac** [comment]
What about remove `rbenv install $(cat .ruby-version)` and the global script from rbenv section? We should ask user to install the right ruby version in the "Installing DEV" step because we'll be sure the user already `cd` into the right directory.

**Comment by Amorpheuz** [comment]
That sounds great, Another change that should be required along with moving Ruby install step after clone, will be to remove the Rails section from `Installing Rails`. The `bundle install` command under `bin/setup` will install Rails 6.0.1 from `Gemfile`.

I will drop the fix with a PR during the weekend, unless a newer contributor would like to pick this up in the meantime.
